src/components/TextInput/TextInput.test.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { render, mount, shallow } from 'enzyme';
+import TextInput from '.';
+
+
+describe('The TextInput component', () => {
+  it('should render without crashing', () => {
+    const wrapper = render(
+      <TextInput
+        onChange={() => {}}
```

Comment:
You can have these props as an object then spread them instead of adding them individually. eg. `<TextInput {...props} />`

Reply:
Noted and refactored.